db/models.py

```diff
@@ -50,3 +58,54 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey("User", on_delete=models.CASCADE)
```

Comment:
Ensure that the `User` model referenced here is correctly linked to the custom user model defined at the end of the file. This requires setting `AUTH_USER_MODEL` in your settings.